docs/docs/cmd/spo/site/site-apppermission-set.md

```diff
@@ -0,0 +1,48 @@
+# spo site apppermission set
+
+Updates a specific application permission for a site
+
+## Usage
+
+```sh
+m365 spo site apppermission set [options]
+```
+
+## Options
+
+`-u, --siteUrl <siteUrl>`
+: URL of the site collection where the permission to retrieve is located
+
+`-i, --permissionId <permissionId>`
```

Comment:
Let's add a remark to `permissionId`, `appId` and `appDisplayName` that we expect users to specify either one of them